script/Ciel/Day 10/10_01_CIEL10_4_2.txt

```diff
@@ -138,12 +138,12 @@ If used properly, it can single-handedly isolate a whole host of Dead Apostles.
 -- Page 196, Offset 27584.
 -- 結界内に閉じこめるには対象を<黒鍵|こっけん>で囲む必要がありますし、大聖堂のような大魔術は連発できないんです。
 You need to surround a target with Black Keys before you can use it to trap them,
- and since the Cathedral is Grand Magecraft it can't be used repeatedly in quick succession.
+ and since the Cathedral is a Grand Magecraft it can't be used repeatedly in quick succession.
```

Comment:
Hmm get Petrikow to clarify countability here

Reply:
@Petrikow a little help please
Also would another comma work here?